product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "New Relic"
+---
+
+The New Relic integration is based on the OpenTelemetry Collector, shipping the metrics over the standard OLTP protocol to the New Relic cloud endpoint.
+
+## Pre-requisites
+
+The New Relic account is required to enable this integration.
+
+## Enable New Relic
+
+To enable the **New Relic** integration: 
+1. Select an existing project from the **Projects** page.
+2. Go to **Settings** on the left-side navigation.
+3. Select **Integrations** from the **Settings** drop down list.
+4. Select **New Relic**, a window pops up to provide the details: 
+   - *New Relic API key**
+   - **New Relic Account ID**
+   - **New Relic API Key Name**
+   Provide all the details and select **Save**.
+
+## Metrics
```

Comment:
For metrics, similar comments to the DD part.

We don't have a comprehensive, stable list yet. Use the same wording as DD about the general set of metrics we send, and explain that these differ from vendor to vendor.

Please warn something like

> The set of metrics delivered to New Relic is subject to change. Metrics whose names begin with `postgres.preview.` or `biganimal.preview.` are very likely to be renamed or removed in a future release. Other metrics may also be renamed, added or removed to better integrate into the New Relic platform.